app/views/admin/facilities/index.html.erb

```diff
@@ -22,7 +22,9 @@
 <% else %>
   <div class="alert alert-info alert-dismissible fade show" role="alert">
     <i class="far fa-question-circle mr-2"></i> A <span class="term">Facility Group</span> is usually a
-    <span class="term">District</span>. First, add a new facility group and then add facilities inside it.
+    <span class="term"><%= t('helpers.label.facility.district').gsub(/\s\*$/, '').strip %></span>.
```

Comment:
Ugh, don't know why we have the `*` in the locale files. It should be in the markup :|

Reply:
Hehe yeah. I chose this surgical approach instead of detangling that right now 😅